.github/workflows/test.yml

```diff
@@ -51,6 +51,8 @@ jobs:
             ruby_version: "2.7"
```

Comment:
How do you feel about also dropping this matrix?

Reply:
I'm ok with that 👍

Reply:
I like the simplicity of having 3 groups, and adding a new one makes me want to drop the old one. I don’t know what sorta social contract we have to the community to always test against 6.1. I say let's go for it cause it lets us move our dependencies forward

Reply:
Decision: leave 6.1 support in as long as it's easy. Remove it as soon as it's inconvenient.